src/01-document/Document.js

```diff
@@ -499,6 +500,10 @@ class Document {
     })
     return this
   }
+
+  options() {
```

Comment:
I'm not thrilled by this code, but I couldn't find a simple way to allow the category plugin to re-use the options that were passed to the original fetch request (which I think it almost definitely what you want in this case).

Reply:
works for me!